tests/collapse.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { test, expect } from '@playwright/test';
+
+test('Collapse E2E Tests', async ({ page }) => {
+  await page.goto('/test.html');
+
+  const collapse = page.getByTestId('collapse')
+
+  // Main title should display
+  const collapseTitleText = 'Robots...'
+  const collapseTitle = collapse.getByText(collapseTitleText)
+  await expect(collapseTitle).toBeVisible()
+  await expect(collapseTitle).toHaveText(collapseTitleText)
+
+  // Slot:title should display
+  const titleSlotText = '(title)'
+  const titleSlot = collapse.getByText(titleSlotText)
+  await expect(titleSlot).toBeVisible()
+  await expect(titleSlot).toHaveText(titleSlotText)
+
+  // Open collapse (by "open" property) should display chevron up
+  const openCollapse = page.getByTestId('collapse-open')
```

Comment:
I would check the visibility of all of your componets (openCollapse, collapse,closedCollapse) at least once in the tests